notification-lambda-cfn.yaml

```diff
@@ -90,4 +93,22 @@ Resources:
       FunctionName: !GetAtt Function.Arn
       Action: lambda:InvokeFunction
       Principal: events.amazonaws.com
-      SourceArn: !GetAtt EventRule.Arn
\ No newline at end of file
+      SourceArn: !GetAtt EventRule.Arn
+
+  LambdaAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmActions: [!Ref AlarmTopic]
+      OKActions: [!Ref AlarmTopic]
+      AlarmDescription: !Sub Triggers if there are errors persisiting on ${Stage}
```

Comment:
I think we need to update this description - we might even want to link to a Google Doc / runbook (using the [magic syntax](https://github.com/guardian/google-chat-bots/pull/51)) with a few tips on how this project works (and what might go wrong / who to speak to if it does)!